mcu_janus.go

```diff
@@ -1063,7 +1063,11 @@ func (p *mcuJanusSubscriber) handleEvent(event *janus.EventMsg) {
 			log.Printf("Subscriber %d: associated room has been destroyed, closing", p.handleId)
 			go p.Close(ctx)
 		case "event":
-			// Ignore events like selected substream / temporal layer.
+			// Handle renegotiations, but ignore other events like selected
+			// substream / temporal layer.
+			if (getPluginStringValue(event.Plugindata, pluginVideoRoom, "configured") == "ok") {
```

Comment:
```suggestion
			if getPluginStringValue(event.Plugindata, pluginVideoRoom, "configured") == "ok" {
```

This should have been caught by the CI linter.

Reply:
Linter is updated in #196

Reply:
_Somebody_ is too used to C-like syntax... ;-) Fixed.